components/pages/articlePageLayout/ArticlePageLayout.tsx

```diff
@@ -18,13 +20,8 @@ function ArticlePageLayout({ pageData }: { pageData: TArticlePageLayoutProps })
 
   const mediaImage = goArticleImage?.mediaImage
 
-  const localeDateString = new Date(
-    pageData.publicationDate.timestamp as string
-  ).toLocaleDateString("da-DK", {
-    day: "numeric",
-    month: "long",
-    year: "numeric",
-  })
+  const publicationDate = new Date(pageData.publicationDate.timestamp)
+  const localeDateString = localeDateStringFromDate(publicationDate)
```

Comment:
I think we should choose a name that reflects the usage of the date. Since this is a date we will make use of throughout he application we could also call it `generalGoDate` or similar.